00_hello_world/main.go

```diff
@@ -9,5 +9,18 @@ import (
 var out io.Writer = os.Stdout
 
 func main() {
-	fmt.Fprintln(out, "Hallo du schöne Welt!")
+
+	fibonacciSeries(7)
+}
+
+func fibonacciSeries(number int) {
+	var a, b = 0, 1
+	fmt.Fprintln(out, a)
+	fmt.Fprintln(out, b)
```

Comment:
It's better to decouple fibonacci sequence calculation from IO. https://github.com/anz-bank/go-training/issues/37